.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```

Comment:
Maybe make the set of 3 buttons self-documenting; they could be labeled eg:
[Open a documentation issue]
[Contact DevTools team]
[Open the WebView2Feedback repo]
The 3 buttons do non-parallel, different modes of actions, so it's confusing if the buttons for both DevTools & WV2 are labeled the same, as "Open".  The word "Open" might be incorrect, in the case of DevTools, it probably means something very different than "Open" in the case of WV2, so it might be best to avoid same label on both those buttons.

Reply:
I'd like to push back on this. As a user who found an issue with Edge, when I go to that issue chooser page, I'm likely to read the titles and the buttons, but less likely to read the (lighter-color) descriptions below them.

If I believe I found a bug, I'll hesitate to "contact" the DevTools team. I just want to report the issue. Not contact someone. 
Similarly, I don't know what opening the WebView2Feedback repo might do. 

I think we have more chances of actually directing people to the right places if the titles read something like: "For an issue about X, go here".

What do you think?

Reply:
Do we already have anything like this? I expect "Report a bug/problem" when I visit sites of any sort.  Bottom of Chrome DevTools doc has Contribute / File a Bug / View Source. I tried to find common ground between 4 or 5 random doc pages. No luck. My findings:

- Our [Devtools extensions for Visual Studio Code](https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension) has Feedback top right. End of article has Send your feedback by filing an issue and contribute to vscode-edge-devtools. Bottom of page has Submit and view Feedback.
- [Chrome devtools](https://developer.chrome.com/blog/new-in-devtools-93/) has end of article section Getting in touch with the Chrome DevTools team. It includes 4 bullet points to Submit feedback via crbug.com, Report DevTools issue with menu path, Tweet to their twitter handle, and Leave comments at their Youtube channel.
- [Postman](https://learning.postman.com/docs/getting-started/installation-and-updates/) has almost nothing until a Postman support link in Next steps at the end of the article.
- [Kubernetes](https://kubernetes.io/docs/tutorials/) has a link to Create and Issue top right of page. The only other mention is at the end of the article: Feedback - Was this page helpful? Yes /No.

My 2 cents. We seem to have a lot of mechanisms for soliciting feedback. Anything we can do to simplify and standardize has my vote. Overwhelming the reader is not helpful.

Reply:
My original comment was suggesting changing the label on the button from "Open" to something else.  Not sure if that was clear.  The UX design does need to take into account about 3 elements for each row: 
* heading
* description sentence
* button label

Reply:
Oh, I was reading it wrong then.  One of the "open"s would be the actual name of the feature / item?

Reply:
GitHub does not let us change the button label. We can only change the heading and description.